src/shared/project-peer-review-item/saveProjectComment.mutation.ts

```diff
@@ -0,0 +1,19 @@
+import graphql from 'babel-plugin-relay/macro';
+import { useMutation } from 'src/relay';
+import { saveProjectCommentMutation } from './__generated__/saveProjectCommentMutation.graphql';
+
+export const useSaveProjectComment = () =>
+  useMutation<saveProjectCommentMutation>(graphql`
+    mutation saveProjectCommentMutation($input: SaveProjectCommentMutationInput!, $projectReviewId: ID!) {
+      saveProjectComment(input: $input) {
+        viewer {
+          projectReview(id: $projectReviewId) {
```

Comment:
With this mutation, the state of person review will be changed and should be fetched.